js/sign/src/cbor/additionalinfo.ts

```diff
@@ -0,0 +1,95 @@
+// 5 bits of the first byte on CBOR contain the additional information which can
+// be either the actual value or a value telling us how to get the value.
+
+// The meaning of additional information depends on the major type. For example,
+// in major type 0, the argument is the value of the data item itself (and in
+// major type 1, the value of the data item is computed from the argument); in
+// major type 2 and 3, it gives the length of the string data in bytes that
+// follow; and in major types 4 and 5, it is used to determine the number of
+// data items enclosed.
+
+// https://www.rfc-editor.org/rfc/rfc8949.html#name-specification-of-the-cbor-e
+
+export enum AdditionalInfo {
+  Direct, // 0->23
+  OneByte, // 24
+  TwoBytes, // 25
+  FourBytes, // 26
+  EightBytes, // 27
+  Reserved, // 28->30
+  Indefinite, // 31
+}
+
+export const convertToAdditionalInfo = (b: number): AdditionalInfo => {
```

Comment:
Let's use the function definition syntax. (Here and elsewhere)
```suggestion
export function convertToAdditionalInfo(b: number): AdditionalInfo {
```